kubernetes/keycloak/README.md

```diff
@@ -1,31 +1,30 @@
-# Building Keycloak
+# Keycloak
 
-Keycloak is built from a modified version of the [dockerfile](https://repo1.dsop.io/dsop/opensource/keycloak/keycloak) published by Ironbank Containers. This Dockerfile is then modified to use the official OpenJDK on Dockerhub. The following changes are made to the dockerfile:
+# Installation
```

Comment:
Should this Installation section be removed? It looks like there is nothing in it?

Reply:
Made requested change